qiskit/circuit/parameter.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, name):
         self._name = name
 
         from sympy import Symbol
-        symbol = Symbol(name)
+        symbol = Symbol(name, real=True)
```

Comment:
Is setting `real=True` required here? This will likely conflict with #5192 .

Reply:
Not required. Resolved.